src/main/java/com/google/codeu/servlets/HomeServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.codeu.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/home")
```

Comment:
If you change this to "/" it should work for http://localhost:8080/. Also once you do this all URLs that don't exist will get routed to this servlet. The comment below will show you how to deal with that.

You should also remove index.html once you have this working.